cachix-api/src/Cachix/Api/Swagger.hs

```diff
@@ -1,3 +1,4 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Orphans should go in their own file to prevent this flag from masking unintended orphans. Also explicit is good.

Reply:
This is the file that only contains orphans.

Reply:
Hehe ok. I would call it `Cachix.Api.Swagger.Orphans`, so you won't accidentally add non-orphan things to it.